client/src/components/Navbar.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from 'styled-components';
+import Dropdown from './DropdownMenu/Dropdown';
+
+const people = [
+  { name: 'Edmon Wong', link: 'https://github.com/Edmond120' },
+  { name: 'Aziz Yokubjonov', link: 'https://github.com/aziz512' },
+  { name: 'Haitham Alnajar', link: 'https://www.google.com' },
+  { name: 'Yankang (Kyle) Xue', link: 'https://github.com/KanG98' },
+  { name: 'Kamille Tipan', link: 'https://github.com/aziz512/youtube-tracker' },
+];
+
+const Navbar = () => {
+  return (
+    <Nav>
+      <Logo href="">
+        Video<span>Tube</span>
+      </Logo>
+      <Menu>
+        <Dropdown people={people}></Dropdown>
+        <MenuLink
+          href="https://github.com/aziz512/youtube-tracker"
+          target="_blank"
+        >
+          Our Work
+        </MenuLink>
+      </Menu>
+    </Nav>
+  );
+};
+
+export default Navbar;
+
+const MenuLink = styled.a`
+  padding: 1rem 2rem;
+  cursor: pointer;
+  text-align: center;
+  text-decoration: none;
+  color: #67bc98;
+  transition: all 0.3s ease-in;
+  font-size: 0.9rem;
+  &:hover {
+    color: #7b7fda;
+  }
+`;
+
+const Nav = styled.div`
+  padding: 0 2rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  background: white;
+  position: absolute;
```

Comment:
there actually doesn't seem to be a good reason to have this as `absolute`, this just complicates the content display (and i'm guessing this is why you added the 30px padding to compensate for it).  
I'm guessing you set it to absolute to make up for the spaces on the left and right but that is better mitigated by restructuring the App.js component to have Navbar outside of Content.

Reply:
I reverted back to the original 10px and removed "position: absolute". I think it should look better now.